assets/src/console/gql/queries.graphql

```diff
@@ -136,6 +139,20 @@ mutation SubmitRefund($paymentId: ID!, $amount: Int!, $reference: String) {
   }
 }
 
+mutation SubmitCardType($gatewayAccountId: ID!, $cardTypeIds: [ID!]!) {
```

Comment:
this should be named something like `UpdateGatewayAccountCardTypes`. the "submit" prefix was specifically terminology we used for refunds rather than for all mutations.

Reply:
Addressed all review comments.